.github/workflows/gke.yaml

```diff
@@ -0,0 +1,120 @@
+name: ConformanceGKE
+
+on:
+  pull_request_target:
+    types:
+      - "labeled"
+  #pull_request: #uncomment for testing
+  #  types:
+  #    - "labeled"
+  push:
+    branches:
+      - master
+
+env:
+  clusterName: cilium-cli-ci-${{ github.run_number }}
+  zone: us-west2-a
+
+jobs:
+  installation-and-connectivitiy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    if: ${{ github.event.label.name == 'ci-run/gke' || github.event_name == 'push' }}
```

Comment:
Should that be a `&&` instead, or am I misunderstanding how `if` works on GH Actions?

I'm thinking we only want to trigger a CI run on push IF the label is present, otherwise on regular pushes we do not want to trigger a run.

Reply:
`push` event is for merging to master, we can debate whether we actually want to run this every time we merge something, but this is working as I intended - either run when `ci-run/gke` label is set or when we are merging to master.